FindSplash.cmake

```diff
@@ -96,7 +102,7 @@ if(Splash_ROOT_DIR)
     #
     find_library(Splash_LIBRARIES
       NAMES splash
-      PATHS $ENV{SPLASH_ROOT}/lib)
+      PATHS ${Splash_ROOT_DIR}/lib)
```

Comment:
yes, but not-found is correct behaviour if the env var is not set and it can not find splash. you have three ways to tell where splash lives if installed in non-default locations:
- export CMAKE_PREFIX_PATH
- cmake CMAKE_PREFIX_PATH
- export SPLASH_ROOT (deprecated)